test/functional/api/rpc_spec.lua

```diff
@@ -0,0 +1,47 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
@bfredl yes, but those are rather unusual in their approach. It can be disorienting for others when a group of tests is using many different techniques.